.github/workflows/android_lint.yml

```diff
@@ -0,0 +1,32 @@
+name: Android Kotlin lint
+
+on: [ pull_request]
```

Comment:
We should run linter and unit tests after merging in develop/main branches as well

Reply:
i will add this steps on merge into develop branch